src/commands/PublishCommand.js

```diff
@@ -81,6 +86,13 @@ export const builder = {
     type: "string",
     requiresArg: true,
   },
+  "prerelease-id": {
```

Comment:
We should only allow `--preid`, like the [`semver` cli](https://github.com/npm/node-semver#usage)